app/views/inthenews/poets.html.erb

```diff
@@ -0,0 +1,84 @@
+<h1 class="sectiontitle"><%= t "inthenews.title" %></h1>
+
+<%= render "inthenews/top_include" %>
+
+<h2>Poets</h2>
+
+<div class="row">
+  <div class="col-md-5">
+  
+    <label for="exampleInput">Search Poets</label>
+    <div class="input-group">
+      <input type="text" class="form-control" placeholder="Search for...">
+      <span class="input-group-btn">
+        <button class="btn btn-default" type="button">Go!</button>
+      </span>
+    </div><!-- /input-group -->
+
+
+    <ul class="timeslider">
+      <li>A</li>
```

Comment:
If you want to be fancy and not have to type things out in the future, you can use a Ruby range:

```
<% ("A".."Z").each do |letter| %>
  <li><%= letter %></li>
<% end %>
```

Reply:
cool! saved for later :D